### ISSUE TITLE
Website SSL Error

### ISSUE THREAD

**Issue by bnonni** [JJTimmons/ff#40]
I normally rely on your app every year for draft day, but I'm now getting this error, and today is my draft day :( bummer. Was going to try to clone and run, but I would need the .env file. Any assistance would be great.

![image](https://user-images.githubusercontent.com/19941207/128599330-018ac613-0648-49bc-a48e-d94cdc92abbe.png)

**Comment by danielisgr8** [comment]
If you're on Chrome, you can type `thisisunsafe` on the error page to bypass this. Not sure of the equivalent for other browsers.

Sorry this won't help you since you already had your draft, but hopefully it can help anyone else that runs into this.

**Comment by JJTimmons** [comment]
Sorry guys, I mucked this year up. A bit distracted w/ work + my league isn't running this year

Really sorry

I just fixed it though (though I realize it's too late to be of use). I switched accounts a year ago (migrated resources from one AWS account to another) but didn't terminate the old CloudFront distribution. That distribution had CNAME records that are globally unique across accounts, so my new account's CF distribution wasn't set up right for the SSL
